tango_simlib/utilities/sim_xmi_parser.py

```diff
@@ -173,17 +173,7 @@ def extract_device_class_descr(self, description_data):
             }
 
         """
-        #class_data = description_data.attrib   # This contains the additional
-                                                # information about the Tango device
-                                                # class, however it is not useful for
-                                                # the current problem.
-        #class_data['identification'] = {}
-        #identification = description_data.find('identification')
-        #class_data['identification']['contact'] = identification.attrib['contact']
-        #class_data['identification']['author'] = identification.attrib['author']
-        #class_data['identification']['emailDomain'] = (
-        #    identification.attrib['emailDomain'])
-        #    class_data['identification'].append(id.attrib)
+
```

Comment:
This change is already in `master`.
See: https://github.com/ska-sa/tango-simlib/pull/103/files